.github/workflows/ci-model-regression.yml

```diff
@@ -561,21 +574,21 @@ jobs:
           fi
 
           if [[ "${{ matrix.type }}" == "nlu" ]]; then
-            poetry run rasa train nlu --quiet -u ${DATASET_DIR}/${DATASET}/${TRAIN_DIR} -c dataset/configs/${CONFIG} --out ${DATASET_DIR}/models/${DATASET}/${CONFIG}
+            poetry run ddtrace-run rasa train nlu --quiet -u ${DATASET_DIR}/${DATASET}/${TRAIN_DIR} -c dataset/configs/${CONFIG} --out ${DATASET_DIR}/models/${DATASET}/${CONFIG}
```

Comment:
Looking at the data that we can get from APM, there are no benefits of using trace + APM in this particular case. I'd remove ddtrace.

Reply:
Or at least I wouldn't enable it by default, rather by an extra-label.